source/includes/common/common-k8s-connect-operator-console.rst

```diff
@@ -1,77 +1,69 @@
+                      "targetPort": 9090,
+                      "nodePort": 30090
+                  },
+                  {
+                      "name": "https",
+                      "port": 9443,
+                      "protocol": "TCP",
+                      "targetPort": 9443,
+                      "nodePort": 30433
+                  }
+              ],
+              "type": "NodePort"
+          }
+      }'
+
+   The patch command should output ``service/console patched``.
+   You can now access the service through ports ``30433`` (HTTPS) or ``30090`` (HTTP) on any of your Kubernetes worker nodes.
+
+   For example, a Kubernetes cluster with the following Operator nodes might be accessed at ``HTTPS://172.18.0.2:30443``:
 
       .. code-block:: shell
-         :class: copyable
-
-         oc minio proxy
-
-   The command output includes a required token for logging into the Operator Console. 
-
-   .. image:: /images/k8s/operator-dashboard.png
-      :align: center
-      :width: 70%
-      :class: no-scaled-link
-      :alt: MinIO Operator Console
 
-   You can deploy a new :ref:`MinIO Tenant <minio-k8s-deploy-minio-tenant>` from the Operator Dashboard.
+         kubectl get nodes -o custom-columns=IP:.status.addresses[:]
+         IP
+         map[address:172.18.0.5 type:InternalIP],map[address:k3d-MINIO-agent-3 type:Hostname]
+         map[address:172.18.0.6 type:InternalIP],map[address:k3d-MINIO-agent-2 type:Hostname]
+         map[address:172.18.0.2 type:InternalIP],map[address:k3d-MINIO-server-0 type:Hostname]
+         map[address:172.18.0.4 type:InternalIP],map[address:k3d-MINIO-agent-1 type:Hostname]
+         map[address:172.18.0.3 type:InternalIP],map[address:k3d-MINIO-agent-0 type:Hostname]
 
-.. dropdown:: NodePorts
-
-   Use the following command to identify the :kube-docs:`NodePorts <concepts/services-networking/service/#type-nodeport>` configured for the Operator Console.
-   If your local host does not have the ``jq`` utility installed, you can run the first command and locate the ``spec.ports`` section of the output.
-
-   .. code-block:: shell
-      :class: copyable
-
-      kubectl get svc/console -n minio-operator -o json | jq -r '.spec.ports'
-
-   The output resembles the following:
+   .. cond:: openshift
 
-   .. code-block:: json
+      Use ``oc`` to temporarily forward the Operator Console port:
 
-      [
-         {
-            "name": "http",
-            "nodePort": 31055,
-            "port": 9090,
-            "protocol": "TCP",
-            "targetPort": 9090
-         },
-         {
-            "name": "https",
-            "nodePort": 31388,
-            "port": 9443,
-            "protocol": "TCP",
-            "targetPort": 9443
-         }
-      ]
+      .. code-block:: shell
+         :class: copyable
 
-   Use the ``http`` or ``https`` port depending on whether you deployed the Operator with Console TLS enabled via :mc-cmd:`kubectl minio init --console-tls`.
+         oc minio proxy
```

Comment:
Isn't this same old `kubectl minio proxy` replacement? May be I missing something.

Reply:
Is that how `oc` does it? 🤔

Reply:
`oc` is a drop-in replacement for `kubectl` in reference of OpenShift.

AFAIK `kubectl` also works in openshift though so I'm not sure we need to worry about `oc` for now.

Reply:
I meant, `oc minio proxy` = `kubectl minio proxy` and this is still using the kubectl minio plugin.

Reply:
Aha - yeah great catch. This entire cond can be dropped.

Reply:
Removed. 👍🏻